Ch03/Ch03.md

```diff
@@ -289,7 +288,7 @@ to our kindergarten.
 ```
 
 
-All of the information is in one place. You can see that we used a semantic ID, `children/`, with the child name as the document ID. This 
+All of the information is in one place. For the document ID you can see that we used a semantic ID, children/alice-liddell, which includes the child name. This 
```

Comment:
`children/alice-liddell` should be using `

Reply:
done